runtime/decimal.c

```diff
@@ -0,0 +1,17 @@
+#include "balrt.h"
+#include "third-party/decNumber/decQuad.h"
+
+static inline decQuad *balDecToDecQuad(GC decimal128 *dp);
+
+TaggedPtr _bal_decimal_const(const char *decString) {
+    GC decimal128 *dp = _bal_alloc(sizeof(decimal128));
+
+    decContext ctx;
+    decContextDefault(&ctx, DEC_INIT_DECQUAD);
+    decQuadFromString(balDecToDecQuad(dp), decString, &ctx);
+    return ptrAddFlags(dp, (uint64_t)TAG_DECIMAL << TAG_SHIFT);
+}
+
```

Comment:
Move this to line 4 so you don't need a declaration.

Reply:
This method was removed